notebooks/official/model_evaluation/automl_video_classification_model_evaluation.ipynb

```diff
@@ -0,0 +1,1391 @@
+{
```

Comment:
<p>Install the packages required for executing this notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/960/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>